__tests__/acceptance/eslint-with-todo-formatter.ts

```diff
@@ -0,0 +1,168 @@
+import execa from 'execa';
+import stripAnsi from 'strip-ansi';
+import { FakeProject } from '../__utils__/fake-project';
+import { readFile as readFixture } from '../__utils__/read-file-cached';
+
+describe('eslint with todo formatter', function () {
+  let project: FakeProject;
+
+  function runEslintWithFormatter(args: string[], options: execa.Options = {}) {
+    const defaultOptions = Object.assign(options);
+    defaultOptions.reject = false;
+    defaultOptions.cwd = options.cwd || project.baseDir;
+
+    return execa(
+      './node_modules/.bin/eslint',
+      ['src', '--format', require.resolve('../..')],
+      defaultOptions
+    );
+  }
+
+  beforeEach(() => {
+    project = FakeProject.getInstance();
+  });
+
+  afterEach(async () => {
+    // await project.dispose();
+  });
+
+  it('should not emit anything when there are no errors or warnings', async () => {
+    project.files = {
+      ...project.files,
+      src: {
+        'no-problems.js': readFixture('with-no-problems.js'),
+      },
+    };
+
+    project.writeSync();
+    project.install();
+
+    const result = await runEslintWithFormatter([]);
```

Comment:
Why don't we default `args` in `runEslintWithFormatter` to `[]` so you don't need to do this?

```suggestion
    const result = await runEslintWithFormatter();
```

Reply:
Yes, I removed it since I realized I was not using it at all.